server/sonar-server/src/main/java/org/sonar/server/rule/index/RuleQuery.java

```diff
@@ -260,12 +260,11 @@ public String getRuleKey() {
     return ruleKey;
   }
 
-  @CheckForNull
   public String getOrganizationUuid() {
```

Comment:
nothing actually prevents this method to return `null`
actually, `new RuleQuery().getOrganizationUuid()` returns `null`